libtextsecure/sendmessage.js

```diff
@@ -563,6 +563,33 @@ MessageSender.prototype = {
     return Promise.resolve();
   },
 
+  async sendContactSyncMessage(contactConversation) {
+    const primaryDeviceKey = window.storage.get('primaryDevicePubKey');
+    const allOurDevices = (await libloki.storage.getAllDevicePubKeysForPrimaryPubKey(
+      primaryDeviceKey
+    ))
+      // Don't send to ourselves
+      .filter(pubKey => pubKey !== textsecure.storage.user.getNumber());
+    if (allOurDevices.length === 0) {
+      return Promise.resolve();
```

Comment:
return a value instead?
c.f. whatever `sendIndividualProto` resolves to (boolean, or null, or nothing?)

Reply:
True, had copied the sendRequestContactSyncMessage function below and then later changed it to be async